citations-impl/impl/src/java/org/sakaiproject/citation/impl/DbCitationService.java

```diff
@@ -553,8 +880,8 @@ protected void deleteCollection(CitationCollection edit)
 			fields[0] = edit.getId();
 
 			boolean ok = m_sqlService.dbWrite(statement, fields);
-			
-          	statement = "delete from " + m_collectionOrderTableName + " where (" + m_collectionTableId + " = ?)";
+
+          	statement = "delete from " + m_collectionOrderTableName + " where (" + m_collectionTableId + " = ? and SECTION_TYPE is NULL)";
```

Comment:
The indentation here was wrong, before, but could you fix it as you're changing the line?